From 6e1e9265b03cf51d09c028d657fe826d7156a783 Mon Sep 17 00:00:00 2001
From: Randy MacLeod <Randy.MacLeod@windriver.com>
Date: Mon, 13 May 2019 17:26:26 -0400
Subject: [PATCH 2/2] valgrind: adjust std_list expected output

The run-ptest script uses a custom --yocto-ptest option
to change the summary statistics that valgrind issues
to a format that matches what ptest-runner expects.

Only this std_list test which checks that strcpy() works
properly on Ubuntu-12.04 and later is affected by this change.
It's normal output when called with --yocto-ptest is a blank
line so adjust the expected output.

Upstream-Status: Inappropriate [Yocto specific]

Signed-off-by: Randy MacLeod <Randy.MacLeod@windriver.com>
---
 drd/tests/std_list.stderr.exp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drd/tests/std_list.stderr.exp b/drd/tests/std_list.stderr.exp
index d18786f80..8b1378917 100644
--- a/drd/tests/std_list.stderr.exp
+++ b/drd/tests/std_list.stderr.exp
@@ -1,3 +1 @@
 
-
-ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
-- 
2.17.0

